### DISCUSSION TITLE
.Net 7 upgrade?

### DISCUSSION THREAD

**Discussion by SeanMartz** [Finbuckle/Finbuckle.MultiTenant#674] [General]
Hi There! 

First - Thank you for this awesome project. 

Second - I was wondering if there was a plan to upgrade to a version 7, with .net 7 support. 

I noticed issues with upgrading `Microsoft.AspNetCore.Identity.EntityFrameworkCore` as well as `Microsoft.Extensions.Identity.Stores` when I was upgrading one of my projects. 

Is this something that you're interested in doing? Would you be interested in a PR with the necessary upgrades?

Thanks Again!

**Comment by AndrewTriesToCode** [comment]
Hi! The current version does target .NET 7. It didn’t break anything so I didn’t bump the major version number. I’m not sure it’s the best or most clear approach so maybe I should jump to version 8 this Fall?

**Reply by AndrewTriesToCode** [reply]
Also what sort of problems did you encounter?

**Reply by SeanMartz** [reply]
Hi There! 

I solved my issue. It was an issue with implicit package installation instead of explicit package installation. 😅 Sorry to bug you! 

I think your approach of waiting for the .net 8 LTS release and upping the version number would be appropriate.


Thanks again for all your work!
